src/pveclib/vec_int128_ppc.h

```diff
@@ -2495,18 +2496,15 @@ vec_clzq (vui128_t vra)
    * After masking we sum across the left and right counts to
    * get the final 128-bit vector count (0-128).
    */
-  vui64_t vt1, vt2, vt3;
-  vui64_t vzero = { 0, 0 };
-  vui64_t v64 = { 64, 64 };
+  vui64_t vt1, vt2, vt3, h64, l64;
+  const vui64_t vzero = { 0, 0 };
 
   vt1 = vec_clzd ((vui64_t) vra);
-  vt2 = (vui64_t) vec_cmplt(vt1, v64);
-  vt3 = (vui64_t) vec_sld ((vui8_t) vzero, (vui8_t) vt2, 8);
-  result = vec_andc (vt1, vt3);
-  result = (vui64_t) vec_sums ((vi32_t) result, (vi32_t) vzero);
-#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
-  result = (vui64_t) vec_sld ((vui8_t) result, (vui8_t) result, 4);
-#endif
+  vt2 = (vui64_t) vec_cmpequd((vui64_t) vra, vzero);
+  vt3 = vec_mrgahd ((vui128_t)vzero, (vui128_t)vt2);
+  h64 = vec_mrgahd ((vui128_t)vzero, (vui128_t)vt1);
+  l64 = vec_and (vt1, vt3);
+  result = vec_addudm (h64, l64);
```

Comment:
Does the comment above this code need to be updated with these changes?

Reply:
The comments are still correct. Just a faster sequence with more instruction parallelism. 
Fixed up the \brief and @param descriptions.